ivadomed/training.py

```diff
@@ -87,8 +87,13 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
             100 - training_params["transfer_learning"]['retrain_fraction'] * 100.))
         old_model_path = training_params["transfer_learning"]["retrain_model"]
         fraction = training_params["transfer_learning"]['retrain_fraction']
+        if 'reset' in training_params["transfer_learning"]:
+            reset = training_params["transfer_learning"]['reset']
+        else :
+            reset = True
```

Comment:
I would recommend having `reset` as mandatory if using the `transfer_learning`: I am just scared this parameter will be forgotten otherwise and yield errors :/ Let's make it explicit I reckon